### PR TITLE
feat: upgrade to fontawesome v6

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] related with #4788
- [x] looks fine, just fixed a little issue regarding icon size

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: styles/global.scss [hunk 1]
```diff
@@ -190,3 +190,8 @@ body {
 hr {
   background: $k-grey;
 }
+
+.icon svg {
+  width: 1rem;
+  height: 1rem;
```
Comment (Jarsen136) [inline styles/global.scss:196]: Why we would like to set a fixed size for every icon 🤔 If so, we have to override the icon size in many places.

**Comment by yangwao** [comment]
what's ENV for pro?

**Comment by yangwao** [comment]
> if that still fails, let's go with the token.

ok, works!

File: styles/global.scss (hunk 1 above)
Comment (Jarsen136) [reply styles/global.scss:195]: Alright. Maybe setting the default size to 1rem would work for most of the icons : ) 

image

Let's bring back this style.
```
.icon svg {
  width: 1rem;
  height: 1rem;
}
```

Comment (Jarsen136) [reply styles/global.scss:195]: I hope so, however, It does not only appear on explore pages but also on other pages : )

image

image

**Review by Jarsen136 — APPROVED** [review]
code lgtm

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 36.68 USD/KSM ~ 1.363 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x55b8e236c947e65286957b67c7b675e53229e3bf6c1f599280080f5c65f0b170](https://kusama.subscan.io/extrinsic/0x55b8e236c947e65286957b67c7b675e53229e3bf6c1f599280080f5c65f0b170)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**